docs/INSTALL-devel.rst

```diff
@@ -0,0 +1,290 @@
+Following guide covers setting up a developement environment for HaaS
+on a CentOS/RHEL based system.
+
+Dependencies: 
+=============
+There are a few things that HaaS expects the operating system to have::
+
+  yum install epel-release bridge-utils  gcc  httpd  ipmitool libvirt \
+  libxml2-devel  libxslt-devel  mod_wsgi net-tools python-pip python-psycopg2 \
+  python-virtinst python-virtualenv qemu-kvm telnet vconfig virt-install
+
+HaaS requires a database server and currently supports only SQLite and PostgreSQL.
+If you choose to use PostgreSQL database it is recommended to create a new system user 
+with a separate home directory. This user will be configured to control the haas database.
+The development environment will be created in its home directory.
+
+Setting PostgreSQL for development environment:
+================================================
+
+If you choose to use sqlite database, skip this section and go to `Getting Started with HaaS Installation`_.
+
+For setting up PostgreSQL, you will have to install the requisite packages on your system.
+Make sure your account can `sudo` to execute the following commands.
+
+Part 1: Install PostgreSQL server. 
+----------------------------------
+
+Initialize the system. Configure PostgreSQL to allow password authentication.
+
+Install the requisite packages on your server::
+
+  sudo yum install postgresql-server postgresql-contrib -y
+
+
+Initialize postgresql::
+
+  sudo postgresql-setup initdb
+
+
+Replace the term **ident** from following lines in file 
+**/var/lib/pgsql/data/pg_hba.conf** with **md5**.
+
+Before::
+
+  # "krb5", "ident", "peer", "pam", "ldap", "radius" or "cert".  Note that
+  host    all             all             127.0.0.1/32            ident
+  host    all             all             ::1/128                 ident
+  #host    replication     postgres        127.0.0.1/32            ident
+  #host    replication     postgres        ::1/128                 ident
+
+After::
+
+  # "krb5", "ident", "peer", "pam", "ldap", "radius" or "cert".  Note that
+  host    all             all             127.0.0.1/32            md5
+  host    all             all             ::1/128                 md5
+  #host    replication     postgres        127.0.0.1/32            ident
+  #host    replication     postgres        ::1/128                 ident
+
+
+Start postgresql service::
+
+  $ sudo systemctl start postgresql
+  $ sudo systemctl enable postgresql
+
+
+Part 2: Create a system user, database and database role.
+---------------------------------------------------------
+
+Setting up development environment with PostgreSQL backend becomes 
+easy with a dedicated user controlling the database as well as the 
+development environment.
+
+Let that username be `haas_dev`.
+For simplicity we will use the same name for database and database role.
+
+Create a new user on your system::
+  
+  useradd -m -d /home/haas_dev -s /bin/bash -c "for HaaS development" haas_dev
```

Comment:
I did not understand that. You mean add two dashes  like change `system` to `--system` ?